pkg/exposer/csi_snapshot.go

```diff
@@ -448,21 +482,24 @@ func (e *csiSnapshotExposer) createBackupPod(ctx context.Context, ownerObject co
 					Name:            containerName,
 					Image:           podInfo.image,
 					ImagePullPolicy: corev1.PullNever,
-					Command:         []string{"/velero-helper", "pause"},
-					VolumeMounts:    volumeMounts,
-					VolumeDevices:   volumeDevices,
+					Command: []string{
+						"/velero",
+						"data-mover",
```

Comment:
Hmm. This doesn't match the design:
```
velero vgdp-ms backup --this-pod xxx --this-container xxx --this-volume xxx --resource-timeout xxx --log-format xxx --log-level xxx
```

It looks like we've replaced the "vgdp-ms" command with "data-mover" -- which is fine, but I think the design should be updated to reflect that. What about the other changes? Do we no longer need to pass in pod and container?

Reply:
@shubham-pampattiwar just pointed out to me that the design referenced in the PR description is the wrong one and that https://github.com/vmware-tanzu/velero/pull/7576 has the correct design -- so this seems to reflect that design properly.

Reply:
Yes, #7576 is the final design, i will correct the design PR number in this PR and its changelog.